examples/openfeature/README.md

```diff
@@ -0,0 +1,45 @@
+<p align="center">
+    <img src="../../logos/openfeature.svg" alt="OpenFeature" width=250 height=250 />
+</p>
+
+# OpenFeature/OpenTelemetry Example
+
+This example shows how you can run Flipt, Jaeger and a example REST API which integrates with Flipt using the [Flipt OpenFeature Provider](https://github.com/flipt-io/flipt-openfeature-provider-go) + [OpenFeature Client SDK](https://github.com/open-feature/go-sdk) + [OpenTelemetry hook](https://github.com/open-feature/go-sdk-contrib/tree/main/hooks/open-telemetry).
+
+!['Jaeger Example'](./jaeger.jpg)
+
+## Example
+
+In this example, we are leveraging Flipt to prototype a new REST API endpoint for our application. This endpoint will be used to return a greeting message to the user based on some value. We will use the OpenFeature Go SDK to integrate with Flipt and the OpenTelemetry hook to send tracing data to Jaeger.
+
+Since this is just a prototype, we will deteministically return a greeting message based on the value of `user` query param in the request. We support 5 languages: English, Spanish, French, Japanese and German, each with a 20% chance of being returned.
```

Comment:
```suggestion
Since this is just a prototype, we will deterministically return a greeting message based on the value of `user` query param in the request. We support 5 languages: English, Spanish, French, Japanese and German, each with a 20% chance of being returned.
```